typings/connection.d.ts

```diff
@@ -299,9 +282,14 @@ export interface TargetTerminusOptions extends BaseTerminusOptions {
  */
 export interface Source extends TerminusOptions {
   /**
-  * @property {number} [distribution_mode] The distribution mode of the link.
```

Comment:
@grs - I had documented this of type "number". I assume that was a typo previously. Can you please confirm whether "distribution_mode" is of type "string".

Reply:
yes it is a string